orc8r/tools/ansible/roles/docker/tasks/install_debian.yml

```diff
@@ -15,6 +15,50 @@
 # Install Docker based on instructions in:
 # https://docs.docker.com/install/linux/docker-ce/ubuntu
 
+# TODO: Remove with next base image release
```

Comment:
Is there a follow up ticket for this?

Reply:
The current [PR about the new base images](https://github.com/magma/magma/pull/14780) already contains these upgrades / removes the TODO.